src/ObjectUtils/SymbolsFileTest.cpp

```diff
@@ -71,4 +71,46 @@ TEST(SymbolsFile, FailToCreateSymbolsFile) {
   EXPECT_THAT(invalid_file, HasError("File does not exist"));
 }
 
+TEST(GetAllowedSymbolFilenamesForModule, ElfFile) {
+  orbit_grpc_protos::ModuleInfo::ObjectFileType object_file_type =
+      orbit_grpc_protos::ModuleInfo::kElfFile;
+  std::filesystem::path directory = std::filesystem::path{"path"} / "to" / "folder";
+  {  // .so file extention
+    const std::vector<std::filesystem::path> file_names =
+        GetAllowedSymbolFilenamesForModule(directory / "lib.so", object_file_type);
+    EXPECT_THAT(file_names, testing::Contains("lib.debug"));
+    EXPECT_THAT(file_names, testing::Contains("lib.so.debug"));
+    EXPECT_THAT(file_names, testing::Contains("lib.so"));
+  }
+
+  {  // generic file extention (.ext)
+    const std::vector<std::filesystem::path> file_names =
+        GetAllowedSymbolFilenamesForModule(directory / "lib.ext", object_file_type);
+    EXPECT_THAT(file_names, testing::Contains("lib.debug"));
+    EXPECT_THAT(file_names, testing::Contains("lib.ext.debug"));
+    EXPECT_THAT(file_names, testing::Contains("lib.ext"));
+  }
+}
+
+TEST(GetAllowedSymbolFilenamesForModule, CoffFile) {
+  orbit_grpc_protos::ModuleInfo::ObjectFileType object_file_type =
+      orbit_grpc_protos::ModuleInfo::kCoffFile;
```

Comment:
Nit: This can be a constant `constexpr ... kObjectFileType`

Reply:
done